cmake/FindRust.cmake

```diff
@@ -175,10 +176,16 @@ function(_corrosion_determine_libs_new target_triple out_libs)
                     set(was_last_framework OFF)
                     continue()
                 endif()
-                # Strip leading `-l` (unix) and potential .lib suffix (windows)
-                string(REGEX REPLACE "^-l" "" "stripped_lib" "${lib}")
-                string(REGEX REPLACE "\.lib$" "" "stripped_lib" "${stripped_lib}")
-                list(APPEND stripped_lib_list "${stripped_lib}")
+                
+                # Flags start with /
+                if (lib MATCHES "^/")
```

Comment:
perhaps we could additionally require here that the `${target_triple}` matches `msvc$` ?